crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1064,7 +1066,7 @@ LVFontGlyphCacheItem *LVFreeTypeFace::getGlyph(lUInt16 ch, lChar16 def_char) {
 
 LVFontGlyphCacheItem* LVFreeTypeFace::getGlyphByIndex(lUInt32 index) {
     //FONT_GUARD
-    LVFontGlyphCacheItem *item = _glyph_cache2.getByIndex(index);
+    LVFontGlyphCacheItem *item = _glyph_cache2.get(index);
```

Comment:
Is this rename voluntarily?
I see you renamed `getByChar` (that you renamed from `get` in 3rd commit) back to `get` - but you did not for `getByIndex`.
(Also, on linux lChar16 = wchar_t = 32 bits on Linux, just like lUInt32 = 32 bits - but may be not on Windows I think - so, somehow, even for glyph index, you could use just one method - may be that's what you aimed at?)